python/_nimblephysics/simulation/World.cpp

```diff
@@ -258,6 +258,17 @@ void World(py::module& m)
             return self->step(_resetCommand);
           },
           ::py::arg("resetCommand"))
+      .def(
+          "integratePositions",
+          +[](dart::simulation::World* self, Eigen::VectorXs initialVelocity)
+              -> void { return self->integratePositions(initialVelocity); },
+          ::py::arg("initialVelocity"))
+      .def(
+          "integrateVelocitiesFromImpulses",
+          +[](dart::simulation::World* self, bool _resetCommand) -> void {
+            return self->integrateVelocitiesFromImpulses(_resetCommand);
+          },
+          ::py::arg("resetCommand"))
```

Comment:
I think `resetCommand` has a default argument. So this should be `::py::arg("resetCommand") = true` (or `false`, I can't remember what the default is)